Add API Docs for 'default_forum_layout' channel data attribute (#5745)
hammerandchisel/discord-api-docs: docs/resources/Channel.md

| default_reaction_emoji?             | ?[default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object | the emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                                           |
| default_thread_rate_limit_per_user? | integer                                                                     | the initial `rate_limit_per_user` to set on newly created threads in a channel. this field is copied to the thread at creation time and does not live update.                                 |
| default_sort_order?                 | ?integer                                                                    | the [default sort order type](#DOCS_RESOURCES_CHANNEL/channel-object-sort-order-types) used to order posts in `GUILD_FORUM` channels. Defaults to `null`, which indicates a preferred sort order hasn't been set by a channel admin  |

> | default_forum_layout?               | ?integer                                                                    | the [default forum layout view](#DOCS_RESOURCES_CHANNEL/channel-object-forum-layout-types) used to display posts in `GUILD_FORUM` channels. Defaults to `0`, which indicates a layout view has not been set by a channel admin  |

Reviewer: When is this null? I thought when it's unset it is 0 and not null

Author: I duped the row above, it's not null, you're correct that it's 0 when a value isn't set. Updated

| LATEST_ACTIVITY | 0     | Sort forum posts by activity                                   |
| CREATION_DATE   | 1     | Sort forum posts by creation time (from most recent to oldest) |

###### Forum Layout Types

| Flag            | Value | Description                                                    |
| --------------- | ----- | -------------------------------------------------------------- |

> | NOT_SET         | 0     | No default has been set for forum channel                      |

Reviewer 2: if it has no default value, how does it display for users by default? list view still?

Reviewer 3: Yes, on the UI it shows as "List" when it's NOT_SET. Because "List" is the current default value.

Reviewer 2: Not necessary, but it could be helpful to include that posts will display as list in this scenario as well

> | LIST_VIEW       | 1     | Display forum channel in List View layout                      |

Reviewer 2: ```suggestion
| LIST_VIEW       | 1     | Display posts as a list                      |
```

> | GALLERY_VIEW    | 2     | Sort forum posts by creation time (from most recent to oldest) |

Reviewer 3: The description is wrong.

Author: copy pasta.. fixed

> | GALLERY_VIEW    | 2     | Sort forum posts by creation time (from most recent to oldest) |

Reviewer 2: ```suggestion
| GALLERY_VIEW    | 2     | Display posts as a collection of tiles |
```

| default_reaction_emoji?             | ?[default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object     | the emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                                | Forum                            |
| default_thread_rate_limit_per_user? | integer                                                                         | the initial `rate_limit_per_user` to set on newly created threads in a channel. this field is copied to the thread at creation time and does not live update.                      | Text, Forum                      |
| default_sort_order?                 | ?integer                                                                        | the [default sort order type](#DOCS_RESOURCES_CHANNEL/channel-object-sort-order-types) used to order posts in `GUILD_FORUM` channels                                               | Forum                            |

> | default_forum_layout?               | ?integer                                                                        | the [default forum layout type](#DOCS_RESOURCES_CHANNEL/channel-object-forum-layout-types) used to display posts in `GUILD_FORUM` channels                                         | Forum                            |

Reviewer: What happens when you set this to null here?

Author: it's not nullable- nice catch, updated
